openshift/template.yaml

```diff
@@ -367,20 +371,20 @@ objects:
 - apiVersion: batch/v2alpha1
   kind: CronJob
   metadata:
-    name: graph-sync-job
+    name: graph-sync
     namespace: ${THOTH_BACKEND_NAMESPACE}
     labels:
       app: thoth-core
   spec:
-    schedule: "@weekly"
+    schedule: "@hourly"
```

Comment:
We will need to check whether we can sync current documents within one hour.

Reply:
In other words - mark already synced documents not to resync again (not the current state).

Reply:
ACK, so we close this PR and I opened https://trello.com/c/oYfZNzSr